scripts/build_images/build_multiplatform.py

```diff
@@ -0,0 +1,86 @@
+import os
+import tempfile
+
+from build import build_images
+from combine import combine_images_from_metadata_dir
+from utils import (
+    buildx_builder,
+    get_current_platform,
+    get_env_or_exit,
+    local_registry,
+    print_and_run_command,
+)
+
+# The container name for the local Docker registry.
+REGISTRY_NAME = "prairielearn-registry"
+
+# The name for the `buildx` builder to create and use.
+BUILDER_NAME = "prairielearn-builder"
+
+
+def get_platforms() -> list[str]:
+    platforms = os.environ.get("PLATFORMS")
+    if platforms is None:
+        return [get_current_platform()]
+
+    # Split the platforms by comma.
+    return platforms.split(",")
+
+
+"""
+Usage example:
+
+```sh
```

Comment:
I actually prefer inline syntax for this.

```
IMAGES=XXX \
.... \
scripts/build.py
```

Reply:
Sure 🤷 done in e761d56.